orttraining/orttraining/models/runner/training_runner.h

```diff
@@ -172,6 +177,14 @@ class TrainingRunner {
   TrainingSession& GetSession() { return session_; }
 
  private:
+  Status PrepareFeedNamesAndFeeds(IDataLoader& training_data_loader,
```

Comment:
TrainingLoop is too fat. We move some code for preparing inputs and outputs for `session_.Run(...)` into these new functions.